app/Entities/FollowEntity.php

```diff
@@ -7,14 +7,16 @@
 
 class FollowEntity implements Arrayable, JsonSerializable
 {
-    private $follower_user_id;
-    private $following_user_id;
+    private $id;
+    private $followerUser;
+    private $followingUser;
 
     public function toArray()
     {
         $array = [];
-        $array['follower_user_id'] = $this->getFollowerUserId();
-        $array['following_user_id'] = $this->getFollowingUserId();
+        $array['id'] = $this->getId();
+        $array['followerUser'] = $this->getFollowerEntity();
+        $array['followingUser'] = $this->getFollowingEntity();
```

Comment:
for array keys let's use snake case instead of camel case. so it can be 
`follower_user` and `following_user`